modules/govuk_search/manifests/monitoring.pp

```diff
@@ -0,0 +1,20 @@
+# == Class govuk_search::monitoring
+#
+# Setup monitoring for a remote Elasticsearch cluster
+#
+class govuk_search::monitoring () {
+
+  $app_domain_internal = hiera('app_domain_internal')
+
+  class { 'collectd::plugin::elasticsearch':
+    es_host  => "elasticsearch5.${app_domain_internal}",
```

Comment:
I think the host should be a parameter, as this will inevitably change to `elasticsearch6` at some point